modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss

```diff
@@ -1,6 +1,9 @@
 @media (min-width: 576px) {
 	.ddm-form-builder,
 	.ddm-translation-manager {
+		.autofit-row > .autofit-col:first-child {
+			width: 100%;
```

Comment:
@marcoscv-work , @eduardoallegrini, @pat270, can you take a look at this? Without changing a lot more... is this the best approach we have?

Reply:
I think we could use `.autofit-col-expand` @brunofariasdeo can you try with your changes?
![ezgif com-gif-maker](https://user-images.githubusercontent.com/46778114/100638972-e8ae8700-3334-11eb-94b6-6a2c3af82e09.gif)

Reply:
:+1: for @eduardoallegrini suggestion